### DISCUSSION TITLE
How should I adjust the training parameters, such as base_anchor_shapes?

### DISCUSSION THREAD

**Discussion by fanhenghui** [Project-MONAI/tutorials#1873] [General]
I am currently researching the detection sub-project. When training for lung nodules smaller than 3mm, how should I adjust the training parameters, such as base_anchor_shapes?

**Comment by KumoLiu** [comment]
Hi @fanhenghui, adjust base_anchor_shapes to include smaller anchor box sizes that correspond to the pixel scale of 3mm nodules in your images. If your image resolution makes 3mm correspond to, say, 5-7 pixels, set anchors to sizes close to these dimensions.

https://github.com/Project-MONAI/tutorials/blob/fc15cddfdbcdcba47be2095c7fb0f85222c3468d/detection/luna16_training.py#L164-L165
